src/Entity/Adherent.php

```diff
@@ -281,8 +282,8 @@ public function getRoles(): array
             $roles[] = 'ROLE_BOARD_MEMBER';
         }
 
-        if ($this->isAdministrator()) {
-            $roles[] = 'ROLE_ADMINISTRATOR';
+        if ($this->isProjectAdministrator()) {
+            $roles[] = 'ROLE_PROJECT_ADMINISTRATOR';
```

Comment:
Same here, I think `ROLE_CITIZEN_PROJECT_ADMINISTRATOR` is more precise.

Reply:
Agreed, fixed. Thx!